core/storage/app_feedback_report/gae_models.py

```diff
@@ -351,59 +424,66 @@ class AppFeedbackReportTicketModel(base_models.BaseModel):
 
     # A name for the ticket given by the maintainer, limited to 100 characters.
     ticket_name = datastore_services.StringProperty(required=True, indexed=True)
+    # The platform that the reports in this ticket pertain to.
+    platform = datastore_services.StringProperty(
+        required=True, indexed=True,
+        choices=PLATFORM_CHOICES)
     # The Github repository that has the associated issue for this ticket. The
     # possible values correspond to GITHUB_REPO_CHOICES.
     github_issue_repo_name = datastore_services.StringProperty(
-        required=False, indexed=True, choices=GITHUB_REPO_CHOICES)
+        required=False, indexed=True,
+        choices=GITHUB_REPO_CHOICES)
     # The Github issue number that applies to this ticket.
     github_issue_number = datastore_services.IntegerProperty(
         required=False, indexed=True)
     # Whether this ticket has been archived.
     archived = datastore_services.BooleanProperty(required=True, indexed=True)
     # The datetime in UTC that the newest report in this ticket was created on,
-    # to help with sorting tickets.
+    # to help with sorting tickets. If the last report was removed from this
```

Comment:
I'm not quite sure what the last sentence means. Could you please elaborate on it a bit more -- what do you mean by "the last report" and "removed"?

Reply:
Added to comment, let me know if this is clearer.